src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.DarcLib.Tests/VirtualMonoRepo/VmrPatchHandlerTests.cs

```diff
@@ -615,6 +616,53 @@ public async Task CreatePatchesWithSubmoduleUrlChangedTest()
         });
     }
 
+    [Test]
+    public async Task PatchIsAppliedOnRepoWithTrailingSlashTest()
+    {
+        //Setup
+        _vmrInfo.Reset();
+        _vmrInfo
+            .SetupGet(x => x.VmrPath)
+            .Returns(VmrPath + "/");
+        _vmrInfo
+            .Setup(x => x.GetRepoSourcesPath(It.IsAny<SourceMapping>()))
+            .Returns((SourceMapping mapping) => VmrPath + "/src/" + mapping.Name);
+
+        _patchHandler = new VmrPatchHandler(
+            _vmrInfo.Object,
+            _localGitRepo.Object,
+            _remoteFactory.Object,
+            _processManager.Object,
+            _fileSystem.Object,
+            new NullLogger<VmrPatchHandler>());
+
+        var patch = new VmrIngestionPatch($"{PatchDir}/test-repo.patch", string.Empty);
+        _fileSystem.SetReturnsDefault(Mock.Of<IFileInfo>(x => x.Exists == true && x.Length == 1243));
+
+        // Act
+        await _patchHandler.ApplyPatch(_testRepoMapping, patch, new CancellationToken());
+
+        // Verify
+        VerifyGitCall(new[]
+        {
+            "apply",
+            "--cached",
+            "--ignore-space-change",
+            "--directory",
+            $"src/{IndividualRepoName}/",
+            patch.Path,
+        },
+        VmrPath + "/"
+        );
```

Comment:
Just a styling nitty comment:

```suggestion
        VmrPath + "/");
```